topics/ecology/tutorials/phylogeny-data-prep/tutorial.md

```diff
@@ -0,0 +1,332 @@
+>        - *"Select species name from a list?"*: `No`
+>            - *"Repeat source species"*: `"Saccharomyces cerevisiae"`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > In principal, all unique clade names occurring in [NCBI taxonomy database](https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html) can be used for species. Capitalization is ignored, multiple words need to bound by apostrophes. Not all "common" English names occur in the taxonomy database. Using Latin names is always safest.
+>    {: .comment}
+>
+{: .hands_on}
+
+## Annotate with Funannotate
+We will predict protein-coding genes from genomic sequences using [Funannotate](https://funannotate.readthedocs.io/) ({% cite Young2019 %}), which collects evidence from different ab-initio gene predictors as well as from RNA-seq or ESTs data. Funannotate has been developed for Fungi but it works with any Eukaryotic genome. The output of Funannotate is a list of ORFs and their translation in GenBank format.
+
+If you would like to learn about genome annotation in more depth, the GTN has a [section]({{ site.baseurl }}/topics/genome-annotation) dedicated to training on genome annotation, including a hands-on tutorial on [Funannotate]({% link topics/genome-annotation/tutorials/funannotate/tutorial.md %}).
+
+> ### {% icon warning %} Slow Step Ahead!
+> Even for a small dataset, Funannotate can take a very long time to run. You can skip this step and use the Genbank files downloaded from Zenodo for the following step. These were generated using Funannotate as described in the hands-on below.
+{: .warning}
+
+> ### {% icon hands_on %} Hands-on: Annotate genome
+>
+> 1. {% tool [Funannotate predict annotation](toolshed.g2.bx.psu.edu/repos/iuc/funannotate_predict/funannotate_predict/1.8.9+galaxy2) %} with the following parameters:
+>    - {% icon param-file %} *"Assembly to annotate"*: `output_masked_genome` (output of **RepeatMasker** {% icon tool %})
+>    - In *"Organism"*:
+>        - *"Name of the species to annotate"*: `Saccharomyces cerevisiae`
+>        - *"Is it a fungus species?"*: `Yes`
+>        - *"Ploidy of assembly"*: `1`
+>    - In *"Evidences"*:
+>        - *"Select protein evidences"*: `Use UniProtKb/SwissProt (from selected Funannotate database)`
+>    - In *"Busco"*:
+>        - *"BUSCO models to align"*: `saccharomycetes`
+>        - *"Initial Augustus species training set for BUSCO alignment"*: `saccharomyces`
+>    - In *"Augustus settings (advanced)"*:
+>        - *"Minimum number of models to train Augustus"*: `15`
+>    
+>
+>    > ### {% icon tip %} Tip
+>    >
+>    > If available, include mRNA and/or ESTs evidence to increase sensitivity of predictions.
+>    {: .tip}
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > When annotating full genomes, increase the number of *'Minimum number of models to train Augustus'* to an appropriate value. For the small sample dataset used here, values larger than 15 will result in failure.
```

Comment:
I would move that just below the "Minimum number of models to train Augustus" parameter